packages/dai-plugin-mcd/test/SavingsService.spec.js

```diff
@@ -83,7 +97,7 @@ describe('Savings Service', () => {
 
     await service.exit(MDAI(2));
     const amountAfterExit = await service.balance();
-    expect(amountAfterExit.toNumber()).toBe(amountBeforeJoin);
+    expect(amountAfterExit.toNumber().toFixed(3)).toBe('1.275');
```

Comment:
Since the DSR is a real value now, `amountAfterExit` will be greater than `amountBeforeJoin`.

Reply:
But how do we know that it will be greater by exactly that amount? If we could see that value calculated in the test rather than hard coded the test will be easier for people to understand.

Reply:
oops, approved before checking on this piece. how's this going?

Reply:
I'm still working on it, actually. In trying to replicate the operation that arrived at this number, it occurred to me that it seems awfully high. I connected with Andrea, who suggested that we might need to call `drip` explicitly in the test to get the right value. That works sometimes, but it returns an even higher number sometimes too, so I'm in the middle of trying to figure out why that's happening.